app/src/main/res/layout/installed_app_item.xml

```diff
@@ -24,9 +24,9 @@
         android:layout_gravity="center"
         android:layout_weight="1"
         android:padding="8dp"
-        android:fontFamily="@font/oxygenregular"
+        android:fontFamily="@font/shabnam"
```

Comment:
If you can, squash this into the previous commit where you made this file.

Reply:
Okay